lib/hackerone/client/report.rb

```diff
@@ -32,7 +32,7 @@ class Report
         medium
         high
         critical
-      ).freeze
+      ).map(&:to_sym).freeze
```

Comment:
Since we've already released a version that supported strings, let's leave these as strings and call a `to_s` during validation. It won't be consistent with what's above but that's fine.